substrate/frame/conviction-voting/src/lib.rs

```diff
@@ -167,10 +167,10 @@ pub mod pallet {
 	#[pallet::event]
 	#[pallet::generate_deposit(pub(super) fn deposit_event)]
 	pub enum Event<T: Config<I>, I: 'static = ()> {
-		/// An account has delegated their vote to another account. \[who, target\]
-		Delegated(T::AccountId, T::AccountId),
-		/// An \[account\] has cancelled a previous delegation operation.
-		Undelegated(T::AccountId),
+		/// An account has delegated their vote to another account.
+		Delegated { who: T::AccountId, target: T::AccountId },
+		/// An account has cancelled a previous delegation operation.
+		Undelegated { who: T::AccountId },
```

Comment:
This change might break things for the clients. It's used in the governance. Not sure what we gain here.
I've seen the issue this PR trying to address. This is my personal opinion. I would not change it for existing events. But for new events I would definitely use named fields.

Reply:
@josepot @Tbaut @ERussel what do you think about such change? do clients handle this with metadata?

Reply:
Thanks for the ping. The benefit it brings compared to the amount of UIs this is potentially going to break, it's indeed not worth it IMHO.